tests/features/featuresSpec.js

```diff
@@ -0,0 +1,3 @@
+// As a big fan of linguistics
```

Comment:
Why is this incomplete spec file being pushed in a commit/branch that deals with CI?